backend/expense/src/main/java/org/egov/digit/expense/util/EnrichmentUtil.java

```diff
@@ -66,7 +71,12 @@ public BillRequest encrichBillForCreate(BillRequest billRequest) {
             billDetail.getPayee().setParentId(billDetail.getBillId());
             billDetail.getPayee().setAuditDetails(audit);
             billDetail.getPayee().setStatus(Status.ACTIVE);
-            
+
+            List<String> Ids = new ArrayList<>();
+            Ids.add(billDetail.getPayee().getIdentifier());
+            Object additionalDetails = genderUtil.getGenderDetails(billRequest.getRequestInfo(),billDetail.getPayee().getTenantId(),Ids);
+            billDetail.getPayee().setAdditionalDetails(additionalDetails);
```

Comment:
@ajay-egov you are replacing the data of additional details without maintaining the existing available data.